resources/delete_from_list.rb

```diff
@@ -35,7 +35,7 @@
       when 1
         case line
         when /#{regexdelim[0]}\s*#{new_resource.entry}/
-          replace_deleted = regexdelim[0] =~ /\s+/ ? regexdelim[0] : ''
+          replace_deleted = regexdelim[0] =~ /\s+/ ? new_resource.delim[0] : ''
```

Comment:
This change fixed the failures. But the check regexdelim[0] =~ /\s+/ probably fails in some edge case. I'll try to come up with some better tests.